### PR TITLE
Penetrator: Support batch adding nodes

### PR DESCRIPTION
**Pull request by 00pf00**
**What type of PR is this?**
/kind feature
**What this PR does**:
Add penetrator component to support batch adding nodes

### CODE REVIEW

File: pkg/penetrator/util/ssh_util.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"golang.org/x/crypto/ssh"
+	"io"
+	"k8s.io/klog/v2"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+func SShConnectNode(ip string, port int, secret conf.JobSecret) (*ssh.Client, error) {
```
Comment (neweyes) [inline pkg/penetrator/util/ssh_util.go:30]: SshConnectNode？

File: pkg/penetrator/util/ssh_util.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	} else {
+		if secret.PassWd != "" {
+			cfg.Auth = []ssh.AuthMethod{ssh.Password(strings.Replace(secret.PassWd, "\n", "", -1))}
+		}
+	}
+
+	return ssh.Dial("tcp", ip+":"+strconv.Itoa(port), cfg)
+}
+
+func FtpUploadFile(ftpClient *sftp.Client, path, file string) error {
```
Comment (neweyes) [inline pkg/penetrator/util/ssh_util.go:56]: What the Ftp does?

File: docs/installation/addnode_via_penetrator_cn.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# 使用penetrator添加边缘节点
+
+## 安装节点
+
+### 通过master节点安装节点
+
```
Comment (neweyes) [inline docs/installation/addnode_via_penetrator_cn.md:6]: More detailed information is expected, such as, working principle, preconditions, how to observe the operation

File: deployment/penetrator.yaml [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+              - penetrator-job
+              - --secret-path=/etc/superedge/penetrator/job/secret/
+              - --jobconf-path=/etc/superedge/penetrator/job/conf/
+              image: ccr.ccs.tencentyun.com/superedge/penetrator-job:v0.1.2
+              imagePullPolicy: IfNotPresent
+              name: job
+              resources:
+                limits:
+                  cpu: 200m
+                  memory: 64Mi
```
[the remark this answers is not in the document]
Comment (00pf00) [reply deployment/penetrator.yaml:322]: It is no problem to specify the installation of 20 nodes in the task

**Review by neweyes — APPROVED** [review]
/lgtm

**Pull request merged**